scripts/makeupdates

```diff
@@ -32,6 +32,9 @@ import tempfile
 RPM_FOLDER_NAME = os.path.expanduser("~/.anaconda_updates_rpm_cache")
 RPM_RELEASE_DIR_TEMPLATE = "for_%s"
 
+# The Python site-packages path for pyanaconda.
+SITE_PACKAGES_PATH = "./usr/lib64/python3.9/site-packages/"
```

Comment:
I would abstract the `python3.9` string to the separate variable to make the switch in the future easier.

Reply:
This is a temporary solution since we have promised to provide a product-independent script that can be used also by Blivet.